Microsoft.Toolkit.Uwp.UI/Media/RadialGradientBrush.cs

```diff
@@ -103,5 +105,26 @@ protected override bool OnDraw(CanvasDevice device, CanvasDrawingSession session
 
             return false;
         }
+
+        private void RegisterAppResumeHandler()
+        {
+            var device = CanvasDevice.GetSharedDevice();
+            device.DeviceLost += RadialGradientBrush_DeviceLost;
```

Comment:
Can't this leak? It will only unassing this when the device is lost.

Reply:
Thanks for letting me know of that, I just fixed that.